hadoop/src/main/java/org/apache/carbondata/hadoop/api/CarbonInputFormat.java

```diff
@@ -159,6 +162,11 @@ public static void setTablePath(Configuration configuration, String tablePath) {
     configuration.set(FileInputFormat.INPUT_DIR, tablePath);
   }
 
+  public static void setCarbonUnmanagedTable(Configuration configuration,
```

Comment:
use boolean to setvalue, not string

Reply:
configuration.set supports string only now. followed like other config set in CarbinTableInputFormat